webportal/src/app/cluster-view/hardware/hardware-detail.component.js

```diff
@@ -19,7 +19,7 @@
 
 const hardwareDetailComponent = require('./hardware-detail.component.ejs');
 const breadcrumbComponent = require('../../job/breadcrumb/breadcrumb.component.ejs');
-const webportalConfig = require('../../config/webportal.config.json');
+const webportalConfig = require('../../config/webportal.config.js');
```

Comment:
Is it a common practice that we'd better 'require' a .js file other than a .json file?

Reply:
@hwuu In fact `require('webportal.config')` is better, it will make webpack to choose the most suitable extension, even `.ts` if exists and related plugin is used.